docs/usage.md

```diff
@@ -27,8 +48,33 @@ calculates a simple ratio between two strings
 
 
 ### partial_ratio
+: r: r
+Calculates the [ratio](#ratio) of the optimal string alignment
+
```

Comment:
What does the r: r mean?

Reply:
This was by mistake. Will push with the updates.